### PR TITLE
Fix lower than constraint

### PR DESCRIPTION
**Pull request by guerrap**

### CODE REVIEW

File: packages/frontend/src/creation-form/constraint-values-form/ranged-values.tsx [hunk 1]
```diff
@@ -43,15 +43,24 @@ export const RangedValuesConstraintForm = ({
         let value0ErrorText = "";
         let value1ErrorText = "";
 
-        if (!value0) {
+        if (
+            type &&
+            type.value === ConstraintType.BETWEEN &&
```
Comment (luzzif) [inline packages/frontend/src/creation-form/constraint-values-form/ranged-values.tsx]: Shouldn't it be `Range` and `NotBetween` too?

File: packages/frontend/src/creation-form/constraint-values-form/single-value.tsx [hunk 2]
```diff
@@ -29,18 +29,19 @@ export const SingleValueConstraintForm = ({
...
 
-        if (!value0) {
+        if (value0 === undefined) {
             valueErrorText = t("error.value0.single.missing");
         } else if (
-            type?.value === ConstraintType.LOWER_THAN &&
-            BigInt(valueErrorText) <= 0n
+            type &&
+            type.value === ConstraintType.LOWER_THAN &&
+            value0 <= 0n
```
Comment (luzzif) [inline packages/frontend/src/creation-form/constraint-values-form/single-value.tsx:37]: This shouldn't really be possible right? There's the `allowNegative` prop that should be set to `false` on all `NumberInput`s and it shouldn't allow users to enter negative numbers. Still, this additional check is probably good to have.

**Review by luzzif — CHANGES_REQUESTED** [review]

File: packages/frontend/src/creation-form/constraint-values-form/single-value.tsx (hunk 2 above)
Comment (guerrap) [reply packages/frontend/src/creation-form/constraint-values-form/single-value.tsx:37]: Ye we can leave the check I guess.

File: packages/frontend/src/creation-form/constraint-values-form/ranged-values.tsx (hunk 1 above)
Comment (guerrap) [reply packages/frontend/src/creation-form/constraint-values-form/ranged-values.tsx]: Fixed in 53e3ec4fb2bd31d742d3ea3941251bece6547356.

**Pull request merged**